resources/css/bem/input-container.less

```diff
@@ -74,12 +71,33 @@
     --input-padding: 15px var(--input-padding-base);
   }
 
+  &--select {
+    --arrow-width: calc(var(--input-padding-base) * 2 + 1em);
+    --input-padding-right: var(--arrow-width);
+    --label-pointer-events: none;
+
+    &::after {
+      .fas();
+      .center-content();
+      content: @fa-var-chevron-down;
+      width: var(--arrow-width);
+      position: absolute;
+      right: 0;
+      bottom: 0;
+      height: 100%;
+      pointer-events: none;
+    }
+  }
+
   &__label {
     display: flex;
     justify-content: space-between;
     margin: 0;
     color: var(--label-colour);
     font-size: @font-size--normal;
-    padding: var(--label-padding-base) var(--input-padding-base) 0;
+    padding: 5px var(--input-padding-base) 0;
+    position: absolute;
```

Comment:
can do with a comment on why it's `position: absolute`, also a note for the sizing of `--label-height` (or calculate it instead?)